lldb/test/Shell/lit.cfg.py

```diff
@@ -179,3 +180,14 @@ def calculate_arch_features(arch_string):
 
 if "LD_PRELOAD" in os.environ:
     config.available_features.add("ld_preload-present")
+
```

Comment:
I guess it fails gracefully, but should this check whether we're on Darwin first?

Reply:
my thinking is that the darwin check is baked into the invocation of xcrun. If you think it's better to check for darwin explicitly, I will make that change it.

Reply:
I think that's obvious to us, but someone not familiar with our platform might wonder why they don't have `xcrun`. I'd throw it behind a check for Darwin.